test/tests/functional/pbs_cpuset.py

```diff
@@ -98,7 +99,7 @@ def test_reliable_job_startup_on_cpuset(self):
 
         # Check mom logs that the launch hook got propagated
         msg = "Hook;launch.PY;copy hook-related file request received"
-        self.mom.log_match(msg, starttime=stime, interval=16, n=10)
+        self.mom.log_match(msg, starttime=stime, interval=16)
```

Comment:
remove big intervals as on a fast machine you will be adding unnecessary delay of 16s. use interval=2 and may be increase max_attempts.

Reply:
Done